api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -259,7 +304,26 @@ public long getQueryBatchSize() {
                 .institutionId(rs.getLong("institution_id"))
                 .institutionalRoleEnum(
                     institutionalRoleFromStorage(rs.getShort("institutional_role_enum")))
-                .institutionalRoleOtherText(rs.getString("institutional_role_other_text")));
+                .institutionalRoleOtherText(rs.getString("institutional_role_other_text"))
+                .highestEducation(educationFromStorage(rs.getShort("education")))
```

Comment:
What format are they expecting these results to be?  I would expect the String version of these enums to be more useful.

Reply:
ok great.  I think I just misunderstood here